libimage/manifest_list.go

```diff
@@ -309,14 +309,14 @@ type ManifestListAddOptions struct {
 	Password string
 }
 
-func (m *ManifestList) parseNameToExtantReference(ctx context.Context, name string, manifestList bool, what string) (types.ImageReference, error) {
+func (m *ManifestList) parseNameToExtantReference(ctx context.Context, name string, manifestList bool, what string, sys *types.SystemContext) (types.ImageReference, error) {
```

Comment:
(Absolutely non-blocking: It’s not universal but the most common parameter order is `Context, SystemContext, others`)